lib/rux-rails/safe_buffer.rb

```diff
@@ -0,0 +1,13 @@
+require 'rux'
```

Comment:
Is this necessary?

Reply:
Probably not…I noticed you had requires in some of the other files so I added it for consistency.